cf-backup/README.md

```diff
@@ -0,0 +1,27 @@
+# Cloudfiles backup image
+
+This image will upload (backup) a directory to cloudfiles every minute using cron and rack.
+
+Building the image:
+
+```
+docker build -t carinamarina/cf-backup cf-backup
+```
+
+Example run:
+
+```
+docker run -e RS_USERNAME='[redacted]' -e RS_API_KEY='[redacted]' -e RS_REGION_NAME='DFW' -e VOLUME='/config' -e CONTAINER='quassel-backup' --volumes-from quassel-data --name backup carinamarina/cf-backup
```

Comment:
Split this command over multiple lines. This is kinda unreadable as is.

Use `--env` instead of `-e`

`--name` should go at the top of the flags then `--volumes-from` then the `--env`s